app_flutter/lib/service/google_authentication.dart

```diff
@@ -18,8 +22,11 @@ class GoogleSignInService extends ChangeNotifier {
       notifyListeners();
     });
 
-    // TODO(chillers): Re-enable when flakiness is fixed. https://github.com/flutter/flutter/issues/52338
-    // _googleSignIn.signInSilently();
+    try {
+      _googleSignIn.signInSilently();
+    } on PlatformException catch (error) {
+      debugPrint(error.message);
+    }
```

Comment:
(You can manually verify this works by opening cocoon in an incognito window with cookies disabled, it should throw the idpiframe PlatformException here)